reference/5-api-publisher/API-Connection-Management.md

```diff
@@ -0,0 +1,64 @@
+| <nobr>`sqlServer`</nobr>         | [Configuration Store for SQL Server](ConfigurationStore/Sql-Server.md)                   | Reads API connection details from and writes successfully published changed version numbers to a SQL Server configuration table. Enables management of individual connection settings with encryption for securely storing keys and secrets using the built-in support for symmetric key encryption in SQL Server.                                                                                                                                                                                                                                                                                        |
+| <nobr>`postgreSql`</nobr>        | [Configuration Store for PostgreSQL](ConfigurationStore/PostgreSql.md)                   | Reads API connection details from and writes successfully published changed version numbers to a PostgreSQL configuration table. Enables management of individual connection settings with encryption support for securely storing keys and secrets using a supplied password for symmetric key encryption using the `pgcrypto` extension for PostgreSQL.                                                                                                                                                                                                                                                 |
+| <nobr>`awsParameterStore`</nobr> | [Configuration Store for AWS Parameter Store](ConfigurationStore/Aws-Parameter-Store.md) | Reads API connection details from and writes successfully published changed version numbers to the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) using the AWS region specified in the `awsParameterStore` property of the _configurationStoreSettings.json_ file. It also assumes the runtime environment has been initialized with the necessary credentials for accessing AWS services. Enables management of individual connection settings with encryption support for securely storing keys and secrets. |
+| `plainText`                      | Plain Text Configuration Store                                                           | Reads connection information defined in plain text in the _plainTextNamedConnection.json file_. Intended for development purposes only. Does not support writing back the successfully published Change Version.                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Tables with a lot of text are hard to read in markdown, but this table actually works - it is relatively easy to read. More importantly, it works well when served as an HTML page, unlike the next table below.

On line 34, I recommend:

```suggestion
| `plainText`                      | Plain Text Configuration Store                                                           | Reads connection information defined in plain text in the `plainTextNamedConnection.json` file. Intended for development purposes only. Does not support writing back the successfully published Change Version.                                                                                                                                                                                                                                                                                                                                                                                          |
```

Reply:
fixed